src/shared/libs/i18n.js

```diff
@@ -79,14 +79,64 @@ const I18N_LOCALE_LABELS = [
     '中文 (繁體) - Chinese (Traditional)',
 ];
 
-const getLocaleFromLabel = (label) => {
+/**
+ * Gets the locale code for a locale label
+ * @param  {string} label Locale label
+ * @return {string}       Locale code
+ */
+export const getLocaleFromLabel = (label) => {
     const languageIndex = I18N_LOCALE_LABELS.findIndex((l) => l === label);
     return I18N_LOCALES[languageIndex];
 };
 
-// Export constants for pre-ES6 compatibility
-module.exports = {
-    I18N_LOCALES,
-    I18N_LOCALE_LABELS,
-    getLocaleFromLabel,
+/**
+ * Gets the locale label for a locale code
+ * Returns 'English (International)' if the locale code does not have a label
+ * @param  {string} locale Locale code
+ * @return {string}        Locale label
+ */
+export const getLabelFromLocale = (locale) => {
+    const languageIndex = I18N_LOCALES.findIndex((l) => l === locale);
+    if (languageIndex === -1) {
+        return I18N_LOCALE_LABELS[0];
```

Comment:
This could break if ordering is changed in `I18N_LOCALE_LABELS`. Can't we define `English (International)` as a default fallback and use that?